### PR TITLE
[bitnami\rabbitmq] Added support of specific service account

### PR DESCRIPTION
**Pull request by aviyam181199**
**Description of the change**

Added the option to choose a specific k8s `ServiceAccount` to run rabbitmq pods with.
It copied from [bitnami\kafka](https://github.com/bitnami/charts/tree/master/bitnami/kafka) chart

**Benefits**

Some k8s environments can give privileged  permissions to service accounts.

**Possible drawbacks**

the name of variable `rbacEnabled` changed to `rbac.create`, it's not backwords compatible but in most of the charts the variable name is `rbac.create` ([bitnami\mariadb](https://github.com/bitnami/charts/blame/master/bitnami/mariadb/README.md#L77), [bitnami\external-dns](https://github.com/bitnami/charts/blame/master/bitnami/external-dns/README.md#L177), [bitnami\kubewatch](https://github.com/bitnami/charts/blame/master/bitnami/kubewatch/README.md#L61) and more.
**Applicable issues**

I didn't opened a issue... I decided to fix it!

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/rabbitmq/templates/role.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.rbacEnabled }}
+{{- if .Values.rbac.create -}}
```
Comment (javsalgar) [inline bitnami/rabbitmq/templates/role.yaml:1]: Please remove the `-` at the end of `}}`

**Comment by aviyam181199** [comment]
I think we should use `serviceaccount.create` and `rbacEnabled`, I can change the new parameter name, `rbac.create`, back to original ,`rbacEnabled` , and I think it would be good for backward competability

**Comment by javsalgar** [comment]
Hi,

Yes, that would make sense

**Comment by javsalgar** [comment]
It fails in our tests because of this

```
Error: template: rabbitmq/templates/statefulset.yaml:45:20: executing "rabbitmq/templates/statefulset.yaml" at <.Values.serviceAccount.create>: nil pointer evaluating interface {}.create
```

**Comment by javsalgar** [comment]
Hi, 

Just pinging to see if you could check the error. If not, we can continue the work ourselves. As you prefer :)

**Pull request merged**